api-reference/beta/resources/webhooks.md

```diff
@@ -37,18 +38,32 @@ Client must store the subscription ID to correlate a notification with the corre
 
 ## Characteristics of subscriptions
 
-You can create subscriptions for resources such as messages, events, contacts, and drive root items.
+You can create subscriptions for resources such as messages, events, contacts, users, groups, and drive root items.
 
 You can create a subscription to a specific folder:
-`https://graph.microsoft.com/beta/me/mailfolders('inbox')/messages`
+`me/mailfolders('inbox')/messages`
 
 Or to a top-level resource:
-`https://graph.microsoft.com/beta/me/messages`
+`me/messages`, `me/contacts`, `me/events`, `users`, or `groups`
 
 Or on a drive root item:
-`https://graph.microsoft.com/beta/me/drive/root`
-
-Creating a subscription in most cases requires read scope to the resource. For example, to get notifications messages, your app needs the `mail.read` permission. Please note that currently the `Files.ReadWrite` permission is required for OneDrive drive root items and drives associated with SharePoint sites require `Files.ReadWrite.All`. 
+`me/drive/root`
+
+Or to a specific id:
+`users/{id}`, `groups/{id}`, `groups('{id}')/conversations`
+
+Creating a subscription requires read scope to the resource. For example, to get notifications messages, your app needs the `Mail.Read` permission. The following table lists the suggested roles needed for each resource.
+
+|Resource/Item |Scope               |
+|--------------|--------------------|
+| Messages     | Mail.Read          |
+| Calendars    | Calendars.Read     |
+| Contacts     | Contacts.Read      |
+| Files        | Files.ReadWrite    |
+| Users        | User.Read.All      |
```

Comment:
same comment as above. The Group.Read.All is sufficient for reading groups, but User.Read.All was not enough for reading users.